source/less/components/_advantages.less

```
@@ -4,9 +4,18 @@
   text-align: center;
   min-width: 320px;
   margin: auto;
+
+  @media (min-width: @tablet-width) {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    min-width: 660px;
+    margin-top: 120px;
+  }
 }
 
 .advantages__text-header {
```

так правильно использовать @media 
```suggestion
.advantages__text-header {
  font-size: 24px;
  font-weight: 300;
  font-style: normal;
  font-stretch: normal;
  letter-spacing: normal;
  line-height: normal;
  text-align: center;
  text-transform: initial;
  color: @white-gray;
  margin: 0;

  @media (min-width: @tablet-width) {
    font-size: 40px;
    padding-top: 59px;
  }

  @media (min-width: @desktop-width) {
    padding-right: 250px;
    padding-left: 250px;
  }

  &_line-two {
    @media (min-width: @tablet-width) {
      padding: 0 40px;
    }

    @media (min-width: @desktop-width) {
      padding-right: 260px;
      padding-left: 260px;
      padding-bottom: 75px;
    }
  }
}
```

---

Ты когда-нибудь это поправишь?